R/utils.R

```diff
@@ -381,3 +384,31 @@ build_datanames_error_message <- function(label = NULL,
     }
   )
 }
+
+disable_teal_tabs <- function(ns, hide_content = TRUE) {
+  shinyjs::disable(selector = sprintf(".teal-body:has('#%s') .nav li a", ns("options_buttons")))
+  if (hide_content) {
+    shinyjs::runjs(
+      sprintf(
+        '$("#%s .tab-content").hide()',
+        ns("tabpanel_wrapper")
+      )
+    )
+  }
+}
```

Comment:
Would this be more understandable (R-wide) if `shinyjs::runjs( ***  .hide())` is replaced with `shinyjs::hide()`?

Reply:
@vedhav WDYT?